service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -262,25 +281,25 @@ private void sendCMOCaseLinkNotifications(final EventData eventData) {
 
     @EventListener
     public void sendNotificationToPartiesAddedToCase(PartyAddedToCaseEvent event) {
-        List<Representative> representatives = event.getRepresentativesToNotify();
         EventData eventData = new EventData(event);
+        Map<String, Object> servedByEmailParameters = partyAddedToCaseContentProvider
+            .getPartyAddedToCaseNotificationParameters(event.getCallbackRequest().getCaseDetails(), EMAIL);
```

Comment:
for consistency eventData.getCaseDetails().getData()can be used as it is used couple lines below

Reply:
will add variable